kuma/static/js/analytics.js

```diff
@@ -1,8 +1,20 @@
 (function(win, doc, $) {
     'use strict';
 
+    // prettier-ignore
```

Comment:
So, this stops the prettier Atom plugin from formatting that specific node of the code. Ideally we would not do `var analytics = mdn.analytics = {....` but instead, simply have

```
var analytics = {
  ...
};

// then here do
mdn.analytics = analytics;
```
Because of that, Prettier formats that line something like:

```
var analytics = (mdn.analytics = {
 ...
});
```

I however did not want to mess with the code to much as part of this PR, so I just added that line so Prettier does not format that part of the code.